docs/technical_reference/costing/ion_exchange.rst

```diff
@@ -19,41 +38,134 @@ The following variables are constructed on the unit block (e.g., m.fs.unit.costi
 .. csv-table::
    :header: "Description", "Symbol", "Variable Name", "Default Value", "Units"
 
-   "description", ":math:`Symbol_{example}`", "variable_name", "1", ":math:`\text{dimensionless}`"
+   "Density of regenerant solution", ":math:`\rho_{regen}`", "``regen_soln_dens``", "1000", ":math:`\text{kg}\text{m}^{3}`"
+   "Regenerant dose required for regeneration per volume of resin [kg regenerant/m3 resin]", ":math:`D_{regen}`", "``regen_dose``", "300", ":math:`\text{kg}\text{m}^{3}`"
+   "Capital cost for one vessel", ":math:`C_{col}`", "``capital_cost_vessel``", "10000", ":math:`\text{\$/}`"
+   "Capital cost for resin for one vessel", ":math:`C_{resin}`", "``capital_cost_resin``", "10000", ":math:`\text{\$/}`"
+   "Capital cost for regeneration solution tank", ":math:`C_{regen}`", "``capital_cost_regen_tank``", "10000", ":math:`\text{\$/}`"
+   "Capital cost for backwash + rinse solution tank", ":math:`C_{bw}`", "``capital_cost_backwash_tank``", "10000", ":math:`\text{\$/}`"
+   "Operating cost for hazardous waste disposal", ":math:`D_{regen}`", "``operating_cost_hazardous``", "10000", ":math:`\text{\$/}\text{year}`"
+   "Regeneration solution flow", ":math:`\dot{v}_{regen}`", "``flow_mass_regen_soln``", "1", ":math:`\text{kg}\text{year}`"
```

Comment:
```suggestion
   "Regeneration solution flow", ":math:`\dot{v}_{regen}`", "``flow_mass_regen_soln``", "1", ":math:`\text{kg}\text{/year}`"
```
Should be kg/yr rather than kg-yr